api/src/main/java/org/pmiops/workbench/monitoring/views/MonitoringViews.java

```diff
@@ -28,7 +28,10 @@
       Aggregation.Sum.create()),
   NOTEBOOK_SAVE("notebook_save", "Save (or create) a notebook"),
   NOTEBOOK_CLONE("notebook_clone", "Clone (duplicate) a notebook"),
-  NOTEBOOK_DELETE("notebook_delete", "Delete a notebook");
+  NOTEBOOK_DELETE("notebook_delete", "Delete a notebook"),
+  WORKSPACE_TOTAL_COUNT("workspace_total_count", "Count of all workspaces (including inactive)"),
```

Comment:
Is the goal of the count metrics to tell if a DB has been dropped? Could we add CONCEPT_SET_COUNT to this then? (Just so we have a complete set) And maybe COHORT_REVIEW_COUNT?

Reply:
These are just a smattering of gauges to showcase the system and its variety. I think they're fun to show off (for example in the Product Dashboard), as well as to cross-reference when something fishy happens (like the number of workspaces goes up 10x in a day).

I'd like for someone else to add the next few gauges and report back on how easy or difficult it was.